src/kernel/type_checker.cpp

```diff
@@ -529,15 +529,15 @@ optional<expr> reduce_native(environment const & env, expr const & e) {
     expr const & arg = app_arg(e);
     if (!is_constant(arg)) return none_expr();
     if (app_fn(e) == *g_lean_reduce_bool) {
-        object * r = ir::run_boxed(env, const_name(arg), 0, nullptr);
+        object * r = ir::run_boxed(env, options(), const_name(arg), 0, nullptr);
         if (!lean_is_scalar(r)) {
             lean_dec_ref(r);
             throw kernel_exception(env, "type checker failure, unexpected result value for 'Lean.reduceBool'");
         }
         return lean_unbox(r) == 0 ? some_expr(mk_bool_false()) : some_expr(mk_bool_true());
     }
     if (app_fn(e) == *g_lean_reduce_nat) {
-        object * r = ir::run_boxed(env, const_name(arg), 0, nullptr);
+        object * r = ir::run_boxed(env, options(), const_name(arg), 0, nullptr);
```

Comment:
@leodemoura FYI, these are the only call sites not supplying the current options